bundle.yaml

```diff
@@ -1,23 +1,23 @@
 series: xenial
 applications: 
   postgresql: 
-    charm: "cs:postgresql-105"
+    charm: "cs:postgresql"
     num_units: 1
     annotations: 
       "gui-x": 1407
       "gui-y": 295.0000305175781
     to: 
       - "0"
   "serial-vault": 
-    charm: "cs:~jamesj/serial-vault-6"
```

Comment:
oh, sorry. I think I was still sleeping when saw this change in green instead of in red. Got it